### PR TITLE
Update

### PR DESCRIPTION
**Pull request by darkdragon-001**
Use [flatpak-external-data-checker](https://github.com/flathub/flatpak-external-data-checker) for automatic PRs on changes.

### CODE REVIEW

**Comment by darkdragon-001** [comment]
bot, build dev.Cogitri.Health

File: dev.Cogitri.Health.json [hunk 1]
```diff
@@ -34,38 +34,52 @@
...
         "env": {
             "CARGO_HOME": "/run/build/Health/cargo",
-            "RUSTFLAGS" : "-L=/app/lib",
+            "RUSTFLAGS": "-L=/app/lib",
             "RUST_BACKTRACE": "1"
         }
     },
-    "modules" : [
+    "modules": [
         {
```
Comment (bilelmoussaoui) [inline dev.Cogitri.Health.json:42]: You could probably drop libadwaita/libsass/sassc as they are included in GNOME SDK 42.

Comment (darkdragon-001) [reply dev.Cogitri.Health.json:42]: Thanks for the hint! Just pushed it without those libraries. Let's see if it builds.

Comment (bilelmoussaoui) [reply dev.Cogitri.Health.json:42]: Make sure it runs as well, just in case ;)

Comment (darkdragon-001) [reply dev.Cogitri.Health.json:42]: nice, works :smile:

**Comment by darkdragon-001** [comment]
@Cogitri Can you have a look?

**Comment by Cogitri** [comment]
Thanks!

**Pull request merged**